src/modules/customer/dtos/update-customer.dto.ts

```diff
@@ -0,0 +1,45 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  IsNotEmpty,
+  IsNumber,
+  IsOptional,
+  IsString,
+  Length,
+  MaxLength,
+} from 'class-validator';
+
+export class UpdateCustomerDto {
+
+  @ApiProperty()
+  @IsString()
+  @IsNotEmpty()
+  @Length(5, 60)
+  public fullName: string;
```

Comment:
All properties inside of `update-customer.dto.ts` should be with `IsOptional` decorator. Because it's not necessary the user will provide these fields to update as mandatory thing. User can provide any combination of properties to update. That's why all properties should be optional in `update-customer.dto.ts`

Reply:
All fields are mandatory at the time of creation, so these all are should be mandatory while updating because when updating a customer record someone can pass empty data in the mandatory field